contour_plot1D.js

```diff
@@ -0,0 +1,118 @@
+class ContourPlot {
```

Comment:
Maybe it would be better to rename both this class and the file to plot1D, it's not really a contour plot in 1D no?